io.github.shiftey.Desktop.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.shiftey.Desktop</id>
+  
+  <name>Github-desktop</name>
+  <summary>GitHub Desktop is an open source Electron-based GitHub app. It is written in TypeScript and uses React</summary>
```

Comment:
```suggestion
  <summary>GitHub Desktop is an open source GitHub app.</summary>
```
Summary doesn't need to tell what this is written in.

Also it would be better to mention this is a fork, since it is.

Reply:
I updated the meta info to explicitly state its a fork